src/app/store.ts

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
+import themeReducer from './features/themeSlice';
+
+export const store = configureStore({
+    reducer: {
+        theme: themeReducer,
+    },
+});
+
+// Infer the `RootState` and `AppDispatch` types from the store itself
+export type RootState = ReturnType<typeof store.getState>;
+// Inferred type: {posts: PostsState, comments: CommentsState, users: UsersState}
```

Comment:
Comment specifies incorrect inferred type. 
Is this comment necessary as more slices can be added to the store in future?